### DISCUSSION TITLE
Delete image

### DISCUSSION THREAD

**Discussion by yuanbin-dev** [chevereto/chevereto#9] [Q&A]
![1](https://user-images.githubusercontent.com/39656612/218642233-b293818e-77d2-4206-9a03-e5bc9a081222.jpg)
![截屏2023-02-14 12 54 30](https://user-images.githubusercontent.com/39656612/218642851-e2a60e18-b025-4704-a4a3-819d5b423ca3.jpg)
![截屏2023-02-14 13 24 48](https://user-images.githubusercontent.com/39656612/218647141-d726bc85-858f-4568-9a57-8d6b62ec0af0.jpg)
The latest version will report an error when I delete the uploaded image. After deletion, it will be blank placeholder

**Comment by rodber** [comment]
Provide error log.

https://v4-docs.chevereto.com/developer/how-to/debug.html

**Comment by yuanbin-dev** [comment]
![13](https://user-images.githubusercontent.com/39656612/218924234-777d140f-3608-4de1-835f-3454e1c84d0a.jpg)

[chevereto-error.log](https://github.com/chevereto/chevereto/files/10739054/chevereto-error.log)

Relevant log has been uploaded

**Comment by rodber** [comment]
This has been fixed on 4.0.8 (next patch). Thanks for reporting.
https://github.com/chevereto/chevereto/issues/8
